content/faq.md

```diff
@@ -40,48 +46,36 @@ title = "Frequently Asked Questions (FAQ)"
 
 * I like VCMI, how can I help to make it better?
 
-> It would be best if you could join [our Slack](https://slack.vcmi.eu/). We mostly need C++ developers, but there are other areas, where we could use some help. Alternatively you can create VCMI mods to try increasing popularity of VCMI platform.
+> It would be best if you could join [our Slack](https://slack.vcmi.eu/). We mostly need C++ developers, QA/testing helpers and native mobile developers for improving mobile ports, but there may be other areas, where we could use some help. Alternatively you can create VCMI mods to try increasing popularity of VCMI platform.
 
 * Is VCMI Mod Design Team (MDT) part of VCMI Team?
 
-> No, they are separate group of modders, but we do some cooperation with them.
-
-* When will be next official release?
-
-> For now we neglect thinking about official releases in favor of focusing on development. Since daily build became available people can play up-to-date version without need of official release.
+> No, they are separate group of modders, but we have some cooperation with them.
 
-* When is VCMI final / 1.0 / 2.0 version going to be released?
+* When is VCMI final / 2.0 / whatever version going to be released?
 
-> Hard to estimate, work on VCMI is distributed between fixing bugs, adding missing features, developing AI, creating new features that do not exist in Heroes 3 etc. We are not close to getting rid of every single bug and having every single missing feature implemented, but if VCMI was just about that then we would probably finish long ago. Extensibility of VCMI engine requires doing various things "better" than original Heroes 3 developers did, for example no doing things easiest way - engine building with extensibility in mind, no hardcoding various mechanics (such as creature abilities). VCMI also provides more robust engine architecture for multiplayer than Heroes 3, but is also harder to develop because of that.
+> Open source projects never stop growing, and they tend to drift in different directions over years. As long as contributors like to add new features, VCMI will continue to evolve. There's no point in waiting for a "final" version. It's hard to estimate any sort of achieving big milestones like "100% original H3 compatibility" since work on VCMI is distributed between fixing bugs, adding missing features, developing AI, creating new features that do not exist in Heroes 3 etc. We are not close to getting rid of every single bug and having every single missing feature implemented, but if VCMI was just about that then we would probably finish long ago. Extensibility of VCMI engine requires doing various things "better" than original Heroes 3 developers did, for example no doing things easiest way - engine building with extensibility in mind, no hardcoding various mechanics (such as creature abilities). VCMI also provides more robust engine architecture for multiplayer than Heroes 3, but is also harder to develop because of that.
```

Comment:
```suggestion
> Open source projects never stop growing, and they tend to drift in different directions over years. As long as contributors like to add new features, VCMI will continue to evolve. There's no point in waiting for a "final" version. It's hard to estimate any sort of achieving big milestones like "100% original H3 compatibility", but you can review our roadmap <https://github.com/vcmi/vcmi/wiki/Roadmap>. Extensibility of VCMI engine requires doing various things "better" than original Heroes 3 developers did, for example no doing things easiest way - engine building with extensibility in mind, no hardcoding various mechanics (such as creature abilities). VCMI also provides more robust engine architecture for multiplayer than Heroes 3, but is also harder to develop because of that.
```